lib/fog/aws/models/compute/server.rb

```diff
@@ -196,6 +198,48 @@ def run_instance_options
           else
             options.delete('SubnetId')
           end
+          if tag_specifications
```

Comment:
I think it might be good to move this processing down into run_instances directly, as this would allow it to be used from either the requests directly or via the model such as this. I think there are also some helper functions for the formatting the might help you, and that you can see a similar formatting challenge with network information here: https://github.com/fog/fog-aws/blob/5b25c70ba14f8243930f6ea4f26599f13cd73162/lib/fog/aws/requests/compute/run_instances.rb#L134

Reply:
Better now ? ;-)

It's still working for me.

Reply:
Yes, thank you, hopefully it was pretty straightforward to move across.